pkg/providers/instancetype/types.go

```diff
@@ -129,6 +129,26 @@ func computeRequirements(ctx context.Context, info *ec2.InstanceTypeInfo, offeri
 		requirements.Get(v1alpha1.LabelInstanceAcceleratorManufacturer).Insert(lowerKabobCase(aws.StringValue(accelerator.Manufacturer)))
 		requirements.Get(v1alpha1.LabelInstanceAcceleratorCount).Insert(fmt.Sprint(aws.Int64Value(accelerator.Count)))
 	}
+	return hardcodeNeuron(requirements, info)
+}
+
+func hardcodeNeuron(requirements scheduling.Requirements, info *ec2.InstanceTypeInfo) scheduling.Requirements {
```

Comment:
Can you add a link to the page where you got this info? 

Can you also add a //TODO: comment above this that says it should be removed when this data is published via DescribeInstanceTypes?